src/applications/vaos/services/slot/index.js

```diff
@@ -6,34 +6,40 @@ import { getAvailableSlots } from '../../api';
 import { mapToFHIRErrors } from '../../utils/fhir';
 import { transformSlots } from './transformers';
 
+/*
+ * This is used to parse the fake FHIR ids we create for organizations
+ */
+function parseId(id) {
+  return id.replace('var', '');
+}
 /**
  * Fetch appointment slots based on start/end date times based on a VistA sites
  * availability for a particular type of care
  *
  * @param {Object} slotsRequest - An object containing the parameters necessary to retrive appointment slots
- * @param {string} slotsRequest.vistaFacilityId 3 digit facility ID
- * @param {string} slotsRequest.vistaTypeOfCareId 3 digit type of care id
- * @param {string} slotsRequest.vistaClinicId 3 digit clinic id
- * @param {string} startDate start date to search for appointments lots formatted as YYYY-MM-DD
- * @param {string} endDate end date to search for appointments lots formatted as YYYY-MM-DD
+ * @param {string} slotsRequest.siteId 3 digit facility ID
+ * @param {string} slotsRequest.typeOfCareId 3 digit type of care id
+ * @param {string} slotsRequest.clinicId clinic id
+ * @param {string} slotsRequest.startDate start date to search for appointments lots formatted as YYYY-MM-DD
+ * @param {string} slotsRequest.endDate end date to search for appointments lots formatted as YYYY-MM-DD
  */
 export async function getSlots({
-  vistaFacilityId,
-  vistaTypeOfCareId,
-  vistaClinicId,
+  siteId,
```

Comment:
I changed some names in here to be more consistent with the rest of the codebase. At some point I want to write up a terminology guide, and switch our code over to using consistent names for the different ids we have.